examples/multi_python_versions/requirements/requirements_lock_3_10.txt

```diff
@@ -1,10 +1,10 @@
 #
-# This file is autogenerated by pip-compile with Python 3.10
+# This file is autogenerated by pip-compile with Python 3.9
```

Comment:
I think this file should be generated with Python 3.10.

Reply:
Done, it seems that the transition had a bug, so this PR will fix 2 bugs at once :D